app/coffee/Errors.coffee

```diff
@@ -0,0 +1,9 @@
+RetryableError = (message) ->
```

Comment:
Maybe use your `overleaf/o-error` package?

Reply:
Given how critical real-time is, I'd rather not do this without more testing. cc @briangough